### PR TITLE
Add .travis.yml and .lint.sh for yaml, shellscripts

### PR DESCRIPTION
**Pull request by viccuad**
This adds simple shellcheck and yamllint linting for now.

### CODE REVIEW

**Comment by viccuad** [comment]
For this to work, someone with enough permissions needs to accept Travis CI's authorization on this repo (see: https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci-using-github).

**Review by satadruroy — APPROVED** [review]
LGTM

File: .lint.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+echo "Linting shell scripts"
+SH_FILES=$(find "$ROOT_DIR" -type f -name '*.sh' -o -name '*.ksh' -o -name '*.bash' | grep -v "shunit2")
+shellcheck --severity=warning $SH_FILES || retcode=1
+
+echo "Linting yamls"
+YML_FILES=$(find "$ROOT_DIR" -type f -name '*.yaml' -o -name '*.yml' | grep -v "shunit2")
+yamllint -d "{extends: relaxed, rules: {line-length: {max: 120}}}" --strict $YML_FILES || retcode=1
+
+if [[ $retcode == 1 ]] ; then
+    echo "Linting failed" && exit 1
```
Comment (satadruroy) [inline .lint.sh:15]: So even a warning with no errors would be considered as failed?

Comment (satadruroy) [reply .lint.sh:15]: oh, nm, it's relaxed.

**Review by satadruroy — APPROVED** [review]

**Pull request merged**